source/Playnite.Common/FileSystem.cs

```diff
@@ -325,5 +325,46 @@ public static void CopyDirectory(string sourceDirName, string destDirName, bool
                 }
             }
         }
+
+        public static string LookupAlternativeFilePath(string filePath, bool errorIfNotFullPath)
+        {
+            return CheckDrivesForPath(filePath, errorIfNotFullPath, path => File.Exists(path));
+        }
+
+        public static string LookupAlternativeDirectoryPath(string directoryPath, bool errorIfNotFullPath)
+        {
+            return CheckDrivesForPath(directoryPath, errorIfNotFullPath, path => Directory.Exists(path));
+        }
+
+        private static string CheckDrivesForPath(string originalPath, bool errorIfNotFullPath, Predicate<string> predicate)

```

Comment:
Can you try/catch the whole method just to be sure? log the error, of course, but also add condition to catch only if debugger is not attached. Thanks

I don't think this should generally fail, but I've seen of lot of weird stuff happening on people's PCs, so just to be sure. Especially if this is just a bonus feature for cases where users use wrong paths in the first place.

Reply:
Do you mean this check when (!PlayniteEnvironment.ThrowAllErrors) or actually !Debugger.IsAttached ?

Reply:
!Debugger.IsAttached